flank-scripts/build.gradle.kts

```diff
@@ -201,4 +201,4 @@ fun <T> withTempFile(block: File.() -> T): T {
     return block(tempFile).also { tempFile.delete() }
 }
 
-tasks["detekt"].dependsOn(tasks["checkIfVersionUpdated"])
+// TODO temporary disabled tasks["check"].dependsOn(tasks["checkIfVersionUpdated"])
```

Comment:
Make a ticket for this so its not forgotten about?

Reply:
yeah I will do it just after merging it